NativeFileDialogs.CppSharpGenerator/NativeFileDialogsLibrary.cs

```diff
@@ -67,7 +79,16 @@ public void Postprocess(Driver driver, ASTContext ctx)
         {
             AddWindowsOnlyAttribute(function);
         }
+
         AddWindowsOnlyAttribute(ctx.FindClass("FilterItemN").First());
+        AddWindowsOnlyAttribute(ctx.FindClass("PickFolderArgsN").First());
+        AddWindowsOnlyAttribute(ctx.FindClass("OpenDialogArgsN").First());
+        AddWindowsOnlyAttribute(ctx.FindClass("SaveDialogArgsN").First());
```

Comment:
Running the generator on my machine, all of these classes seem to just not exist. I had to comment them out for it to work.

Reply:
Maybe they got removed after an update

Reply:
They got added in this update.

Reply:
Aha. I think these types are actually typedefs instead of classes on Linux. I spotted some `PickfoldernargsT` typedef being processed in there. You might need to make sure both classes and typedefs are being modified. I don't see anything resembling `RenameTargets.Typedef` though...

Reply:
> Running the generator on my machine, all of these classes seem to just not exist. I had to comment them out for it to work.

I guess this is a limitation of CppSharp?

Reply:
I'm okay with just removing them. My interest is primarily in the new `pick folder multiple` option.

Reply:
> I guess this is a limitation of CppSharp?

If there's a limitation, it would be the inability to generate bindings for cross-platform code that has special cases for a different OS... But maybe we're just doing it wrong :/

Reply:
I think all type defs just get replaced with the underlying type.